docs/source/apidoc/core.rst

```diff
@@ -38,6 +38,7 @@ register can be defined at build time.
 
```

Comment:
I noticed there's a typo in the paragraph above (hardware **provider** instead of "provided"). Can you change it in this PR?

Reply:
Sure !